harness/determined/cli/workspace.py

```diff
@@ -422,6 +464,27 @@ def yaml_file_arg(val: str) -> Any:
                     cli.Arg("--json", action="store_true", help="print as JSON"),
                 ],
             ),
+            cli.Cmd(
+                "bindings",
```

Comment:
A workspace can create projects and a resource pool may bind to workspace(s).
* To create a project: `det project create workspace_name name`
* To bind resource pool: `det resource-pool binding add pool_name workspace_names`

I'm trying to figure out the best way to bind a namespace to workspace.
* Do we want to stick to the convention above, like:
    * `det cluster binding add cluster_name workspace_name --auto-create-namespace`
    * `det cluster binding add cluster_name workspace_name --namespace_name name`
* Or the existing commands work better:
    * `det workspace bindings add workspace_name cluster_name --auto-create-namespace`
    * `det workspace bindings add workspace_name cluster_name --namespace_name name`

Let me get back to you on this.

Reply:
@gt2345, @salonig23 and I talked a good amount about the best convention to follow for creating commands that perform workspace-namespace bindings. I think we settled on `det workspace bindings add workspace_name cluster_name --auto-create-namespace` and `det workspace bindings add workspace_name cluster_name --namespace_name name` because unlike resource pools, namespaces aren't determined concepts. Therefore, Kubernetes should be the source of truth for all metadata pertaining to a given namespace. All that we want to store on Determined's side is the workspace-namespace bindings, which is why we settled on adding the command to the set of workspace-related commands. 
Starting the command with `cluster` instead of `workspace` would also imply that we would want to hit a `cluster` related API endpoint, which kind of goes against the organizational flow of adding metadata to our workspaces. 
WDYT @jgongd @salonig23  @gt2345